### ISSUE TITLE
Impossible to install with `$RUSTC_WRAPPER` set

### ISSUE THREAD

**Issue by TriplEight** [LukeMathWalker/cargo-chef#12]
I've [`sccache`](https://github.com/mozilla/sccache/) installed and use it actively.
Unfortunately, `cargo-chef=v0.1.6` (and v0.1.5, didn't dig deeper) throws an error when trying to
```
cargo install cargo-chef
```

<details>
<summary>Error:</summary>

```
info: cleaning up downloads & tmp directories
 ~ cargo install cargo-chef                                                                                                                                                                                                  
    Updating crates.io index
  Downloaded cargo-chef v0.1.6
  Downloaded 1 crate (13.1 KB) in 1.09s
  Installing cargo-chef v0.1.6
error: failed to compile `cargo-chef v0.1.6`, intermediate artifacts can be found at `/tmp/cargo-installWWQz80`

Caused by:
  failed to run `rustc` to learn about target-specific information

Caused by:
  process didn't exit successfully: `sccache rustc - --crate-name ___ --print=file-names --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=cfg` (exit code: 2)
  --- stderr
  error: Connection to server timed out
```

</details>

Workaround:
`unset RUSTC_WRAPPER` solves the problem.

**Comment by LukeMathWalker** [comment]
I managed to reproduce the problem, but honestly I have no clue on how to solve it :sweat_smile: 
`cargo-chef`, as far as I can see, is not doing anything peculiar that could justify a build failure. 
Out of curiosity, I tried to install a bunch of other `cargo` subcommands and they all fail to install more or less with the same error message: `cargo-edit`, `cargo-tarpaulin`, etc.

I would suggest opening an issue on `sccache`'s repository.
